arch/configure.defaults

```diff
@@ -798,6 +798,30 @@ CPPFLAGS            = -D_UNDERSCORE -DBYTESWAP -DLINUX -DIO_NETCDF -DIO_BINARY -
 ARFLAGS             =
 CC_TOOLS            =
 
+########################################################################################################################
+#ARCH    Linux x86_64, Intel oneAPI compiler with icx   # serial serial_NO_GRIB2 dmpar dmpar_NO_GRIB2
+#
+COMPRESSION_LIBS    = CONFIGURE_COMP_L
+COMPRESSION_INC     = CONFIGURE_COMP_I
+FDEFS               = CONFIGURE_FDEFS
+SFC                 = ifx
+SCC                 = icx
+DM_FC               = mpif90 -f90=$(SFC)
```

Comment:
If there's not a good reason for doing so, I think it would be best to not add `-f90=$(SFC)` (and similarly for `-cc=$(SCC)`) as an argument, as I'm not sure this is a standard argument that's accepted by all MPI wrappers.

Also, I would propose switching from `mpif90` to `mpifort`.